app/controllers/pageflow/entries_controller.rb

```diff
@@ -19,14 +19,25 @@ class EntriesController < Pageflow::ApplicationController
     helper RenderJsonHelper
 
     def index
-      theming = Theming.for_request(request).with_home_url.first!
+      respond_to do |format|
+        format.html do
+          @theming = Theming.for_request(request).with_home_url.first!
+          redirect_to(@theming.home_url)
+        end
 
-      redirect_to(theming.home_url)
+        # Atom feed is only available for accounts with a CNAME.
+        format.atom do
+          @theming = Theming.for_request(request).first!
+          @account = @theming.account
+          @entries = EntryFeed::Scope.new(@account.entries, params[:page]).entries
+        end
+      end
     end
 
     def show
       respond_to do |format|
         format.any(:html, :css) do
+          @theming = Theming.for_request(request).with_cname.first
```

Comment:
The default [theming request scope used by `Theming.for_request`](https://github.com/codevise/pageflow/blob/6e500d5c821d06764ae6b4f4d52c6d717cf86465/app/models/pageflow/cname_theming_request_scope.rb#L5) already only matches themings with a CNAME. Dropping the `with_cname` would allow using feeds with custom theming request scopes (e.g. Hosted Pageflow supports looking up themings via domains of the form `account-name.pageflow.io`).